routes/accounts.js

```diff
@@ -270,6 +270,44 @@ function accountRoutes (server, options, next) {
     }
   }
 
+  var postAccountsSessionRoute = {
+    method: 'POST',
+    path: '/accounts/{id}/sessions',
+    config: {
+      auth: false,
+      validate: {
+        headers: validations.sessionIdHeader,
+        query: validations.accountQuery,
+        payload: validations.accountPayload,
+        failAction: joiFailAction
+      }
+    },
+    handler: function (request, reply) {
+      var username = request.payload.data.attributes.username
+      var password = request.payload.data.attributes.password
+      var query = request.query
+
+      var sessionId = toSessionId(request)
+
+      admins.validateSession(sessionId)
+
+      .then(function (_) {
```

Comment:
if you don’t do anything with the argument, leave it out entirely (`.then(function () {`) or comment it out (`.then(function (/* doc */) {`)